core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -164,6 +169,16 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
     }
   }
 
+  /**
```

Comment:
This is just copy & paste of the parent class's doc, and doesn't even reflect what it actually does (since no threads are being started, for example).